lib/custom-resource-handlers/src/certificate-signing-request.ts

```diff
@@ -60,12 +60,12 @@ async function _createSelfSignedCertificate(event: cfn.Event): Promise<ResourceA
     const configFile = await _makeCsrConfig(event, tempDir);
     const pkeyFile = await _retrievePrivateKey(event, tempDir);
     const csrFile = path.join(tempDir, 'csr.pem');
-    await _exec('/opt/openssl', 'req', '-config', configFile,
+    await _exec('openssl', 'req', '-config', configFile,
```

Comment:
is it due to any version change, that the path is modified now or it could've been this way earlier as well

Reply:
I found that openssl is installed in /bin, but in general we do not need to determine the full path